w3cdatalayer/application/factory.go

```diff
@@ -302,6 +311,10 @@ func (s Factory) getProductCategory(product productDomain.BasicProduct) *domain.
 func (s Factory) getProductInfo(product productDomain.BasicProduct) domain.ProductInfo {
 	baseData := product.BaseData()
 	retailerCode := baseData.RetailerCode
+
+	productName := baseData.Title
```

Comment:
This statement is redundant. productName is immediately overwritten in next statement.